src/main/java/com/orbitz/consul/config/ClientConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.orbitz.consul.config;
+
+import com.google.common.base.Preconditions;
+
+public class ClientConfig {
+
+    private CacheConfig cacheConfig;
```

Comment:
This could be private

Reply:
I do not get your point here. It is already private.
Did you refer to another location?